package.json

```diff
@@ -17,7 +20,7 @@
     "access": "public"
   },
   "scripts": {
-    "dev": "nuxt test/fixture",
+    "dev": "nuxt test/fixture/bot-block",
```

Comment:
Why changing fixtures to a specific feature? Can't we add this to the main fixture to skip bots?

Reply:
If you want it to block by default, sure I can change this to 'basic'.  I didn't think it mattered that much (shrug)